src/linter/block.go

```diff
@@ -1775,6 +1783,60 @@ func (b *BlockWalker) handleAssign(a *ir.Assign) bool {
 	return false
 }
 
+func (b *BlockWalker) handleAssignOp(o ir.Node) (res bool) {
```

Comment:
`o` is probably not the best name for the assignment node.
It's assignment+op, but the main part is assignment, not op.